search/src/App.js

```diff
@@ -13,6 +13,23 @@ import './App.css';
 
 class App extends Component {
 	render() {
+		let facetProps = {
+			queryFormat: "and",
```

Comment:
Actually, does this belong in `multilistFacetProps`?
I think practically users will want multilist to be treated as "or" but the behaviour is pretty surprising.